iphone/cli/commands/_buildModule.js

```diff
@@ -894,8 +917,6 @@ iOSModuleBuilder.prototype.runModule = function runModule(next) {
 
 			// 5. unzip module to the tmp dir. Use native binary on macOS, as AdmZip doesn't support symlinks used in mac catalyst frameworks
 			const proc = spawn('unzip', [ '-o', this.moduleZipPath, '-d', tmpProjectDir ]);
-			proc.stdout.on('data', data => this.logger.info(data.toString().trimEnd()));
```

Comment:
without these lines, the unzip never finished for me. We have to "consume" the stdout/stderr for it to finish.